src/qt/diagnosticsdialog.cpp

```diff
@@ -325,7 +610,10 @@ void DiagnosticsDialog::clkFinished()
 {
     if (udpSocket->waitForReadyRead())
```

Comment:
I wonder if we can reduce the timeout here for the clock (the default is 30 seconds): 

```suggestion
    if (udpSocket->waitForReadyRead(10 * 1000))
```

Reply:
I was seeing instances where it would not timeout at all. My guess is that it was "hanging" before that.

Reply:
I put that in anyway.